src/main/java/com/pingidentity/spring/example/config/OAuth2LoginSecurityConfig.java

```diff
@@ -14,6 +15,7 @@
 import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
 
 @Configuration
+@EnableWebSecurity
```

Comment:
Could u please specify the reason of adding `@EnableWebSecurity` explicitly ?

Reply:
This declaration is consistent with the Spring official docs. In this case, annotation truly does nothing and is optional.
https://docs.spring.io/spring-security/site/docs/current/reference/html/jc.html#hello-web-security-java-configuration

Reply:
On previous versions if you define a @Configuration with @EnableWebSecurity spring will switch off the default webapp security settings in Spring Boot, but leave the Actuator’s security enabled. That's  wasn't in our sample requirements, thereby we didn't include it.
Right now [documentation](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle//) doesn't tell much about it.

> The annotation is only optional if we're just overriding the default behavior using a WebSecurityConfigurerAdapter. 

is mentioned [here](https://www.baeldung.com/spring-boot-security-autoconfiguration).

So I would rather check wht this annotation does right now.
Please note here ur findings so we could decide whether to add it or not

Reply:
I have checked - everything works as expected. But I'll remove it as we don't use any provided functionality.